pkgs/servers/baserow/default.nix

```diff
@@ -122,8 +124,9 @@ buildPythonPackage rec {
     twisted
     unicodecsv
     uvicorn
+    watchgod
     zipp
-  ];
+  ] ++ uvicorn.optional-dependencies.standard;
```

Comment:
This only gets you the optional dependencies, but not uvloop itself. And `uvloop[standard]` requests both.